bika/lims/browser/analysisrequest/analysisrequests.py

```diff
@@ -743,35 +751,35 @@ def __init__(self, context, request):
                  'title': _('Print stickers'),
                  'url': 'workflow_action?action=print_stickers'}],
              'columns': ['Priority',
```

Comment:
Transitions `publish`, `republish`, `verify` and `reinstate` will never be available for ARs displayed when using this `unassigned` filter: only `active` (so `reinstate` will never be used) and those ARs that at least have one Analysis with result pending will be displayed (so `verify`, `publish` and `republish` will never be used).